packages/elements/src/combo-box/__test__/combo-box.value.test.js

```diff
@@ -132,5 +132,19 @@ describe('combo-box/Value', function () {
       el.value = '';
       expect(el.value).to.equal('', 'Value must be empty string when reset value on free text mode');
     });
+    it('Cannot set any value without free text mode when data is empty', async function () {
+      const el = await fixture('<ef-combo-box lang="en"></ef-combo-box>');
+      el.value = 'AL';
+
+      await elementUpdated(el);
+      expect(el.value).to.equal('');
+    });
+    it('Set any value with free text mode when data is empty', async function () {
```

Comment:
It seems this test suite breaks our "it should" convention. Let's gradually fix this be applying the convention to new test cases.
```suggestion
    it('Should update value in free text mode when data is empty', async function () {
```